source/pages/albedo.ipynb

```diff
@@ -0,0 +1,358 @@
+{
```

Comment:
<p>Do you know why there is the staircase like behavior? Due to lack of decimals?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AssessingSolar/unofficial-psm3-userguide/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
Yep, the API returns albedo values rounded to two decimal places